src/overlays/actors/ovl_En_Rz/z_en_rz.h

```diff
@@ -2,14 +2,47 @@
 #define Z_EN_RZ_H
 
 #include "global.h"
+#include "assets/objects/object_rz/object_rz.h"
+
+#define EN_RZ_GET_SISTER(thisx) ((thisx)->params & 0x8000)
+#define EN_RZ_GET_TYPE(thisx) ((thisx)->params & 0xF)
+#define EN_RZ_GET_PATH(thisx) (((thisx)->params & 0x7E00) >> 9)
+
+typedef enum {
+    /* 0 */ EN_RZ_JUDO, // in red
+    /* 1 */ EN_RZ_MARILLA // in blue
+} RosaSisters;
```

Comment:
```suggestion
typedef enum {
    /* 0 */ ROSA_SISTERS_JUDO, // in red
    /* 1 */ ROSA_SISTERS_MARILLA // in blue
} RosaSistersSister;
```
Since you name the enum `RosaSisters` that should probably be the prefixes used in the enum. Also since `RosaSisters` is the prefix, should add what it referencing. In this case it is the sister.